<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Projeler</class>
 <widget class="QDialog" name="Projeler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="excel_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>40</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ek - Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sil_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="sonuc_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>761</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="projeler_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="degistir_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Değiştir</string>
   </property>
  </widget>
  <widget class="QLabel" name="siparis_sayisi_lbl">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sipariş Sayısı  : 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="siparis_edilen_lbl">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sipariş Edilen : 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="kalan_siparis_lbl">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>420</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kalan Sipariş  : 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
